pkg/blockdevice/v1alpha1/blockdevice.go

```diff
@@ -289,6 +291,19 @@ func filterNonSparseDevices(originalList *BlockDeviceList) *BlockDeviceList {
 	return filteredList
 }
 
+func filterNonRelesedDevices(originalList *BlockDeviceList) *BlockDeviceList {
```

Comment:
Let us write a comment for this function. Is this supposed to return a list of BDs whose claimstate is "Released" ?

Reply:
It will return BDs whose claimstate is other than `Released`. Will add a comment in next PR